src/module/data/actor/base.mjs

```diff
@@ -46,6 +42,19 @@ export default class BaseActorModel extends foundry.abstract.TypeDataModel {
     return schema;
   }
 
+  /**
+   * Helper function to fill in the `biography` property
+   * @returns {Record<string, foundry["data"]["fields"]["StringField"]}
+   */
+  static actorBiography() {
+    const fields = foundry.data.fields;
+    return {
+      value: new fields.HTMLField(),
+      gm: new fields.HTMLField(),
+      languages: new fields.SetField(new fields.StringField({blank: true, required: true}))
```

Comment:
Will there be specific choices here later?

In the meantime, consider adding the `slug: true` property.

Reply:
I've kinda come around to "Choices will be implemented by passing `options` in the sheet forms to avoid the issue where disabling a module that adjusts CONFIG creates a bunch of invalid documents"

I do also have #60 as its own ticket.